Singletons

```diff
@@ -0,0 +1,40 @@
+# Singletons
+
+If a singleton must be used, then they should always be implemented as a class with a shared reference to an instance of itself, with instance properties, rather than static properties. i.e.
+
+```swift
+// Good
+class MyManager {
+  static let shared = MyManager()
+  var someVar: String = "Hello"
+}
+
+// Bad
+class MyManager {
+  static var someVar: String = "Hello"
```

Comment:
Could we have a different good / bad example and merge the good / bad example with the one below? 
I think there's value in having a static property and it is useful in certain cases (e.g. class identifier) and in that case a singleton is not needed.

Reply:
I'm not really following? I agree that static properties are useful (and indeed required) in certain cases. I think this is clear though that it only applies to singletons, and not to static properties in general?